rose/learner.py

```diff
@@ -346,3 +346,159 @@ def _parallel_active_learn():
 
         # block/wait for each workflow until it finishes
         [learner.result() for learner in submitted_learners]
+
+
+class ParallelActiveLearningAlgoSelector(ActiveLearner):
```

Comment:
The name of the class is very long. I would suggest making the name of this class something like:
`AlgorthimSelector` as the main job of this class is `Selecting The best algorithm.`